domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2582,10 +2584,11 @@ public ResultsHelper(SQLRowStream sqlRowStream, Class<T> clazz) {
    * @param columnNames
    * @param clazz
    * @param isAuditFlavored
+   * @param externalColumnSetters
    * @return
    */
-  <T> Map<String, Method> getExternalColumnSetters(List<String> columnNames, Class<T> clazz, boolean isAuditFlavored) {
-    Map<String, Method> externalColumnSetters = new HashMap<>();
+  <T> void getExternalColumnSetters(List<String> columnNames, Class<T> clazz, boolean isAuditFlavored,
```

Comment:
Method name getExternalColumnSetters is misleading because it doesn't return anything.
Can you replace it by a better method name, for example collectExternalColumnSetters or setupExternalColumnSetters?

Reply:
Sure